README.md

```diff
@@ -24,6 +22,17 @@ This library targets the .NET Standard version 2.0. For an updated list of suppo
 platforms and compatibility with projects that target the PCL, please consult
 [Official Microsoft's Documentation on .NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard).
 
+# Versioning and Compatibility
+
+Although omise-dotnet can be used with other .NET versions, the versions listed below have been test-validated in our CI pipeline to ensure compatibility and stability.
+
+| `omise-dotnet` Version | Supported Omise API Version | .NET Version(s) Supported                             | Type of Support         | Notes                                                                                              |
```

Comment:
- How about renaming `omise-dotnet Version` to `Library Version` or just `Version` and `Type of Support` to just `Support`?
- Moving `Support` next to `Version` would be more clear indicating that support refers to library support not API version support.